src/main/java/com/flightstats/hub/dao/aws/writeQueue/WriteQueue.java

```diff
@@ -0,0 +1,8 @@
+package com.flightstats.hub.dao.aws.writeQueue;
+
+import com.flightstats.hub.model.ChannelContentKey;
+
+public interface WriteQueue {
+    boolean add(ChannelContentKey key);
```

Comment:
Related to my previous comment on default methods, I think if "return true" was the default method here, the NoOpWriteQueue can just go away and where it was used you'd instead just do ```new WriteQueue(){}```;

Reply:
Agreed, the short classes are useful for readability. The default behavior in the interface still might make sense.